langchain/retrievers/aws_kendra_index_retriever.py

```diff
@@ -1,95 +0,0 @@
-"""Retriever wrapper for AWS Kendra."""
-import re
-from typing import Any, Dict, List
-
-from langchain.schema import BaseRetriever, Document
-
-
-class AwsKendraIndexRetriever(BaseRetriever):
```

Comment:
happy to let you make this call, but is there any argument to be made to keep this around and add a deprecation warning, for backwards compatibility purposes?

Reply:
could even make this inherit from AmazonKendraRetriever and in init just map init var names to super() as needed (and get rid of all the methods on this class)